realis.go

```diff
@@ -195,9 +199,9 @@ func BackOff(b Backoff) ClientOption {
 	}
 }
 
-func InsecureSkipVerify(InsecureSkipVerify bool) ClientOption {
+func insecureSkipVerify(insecureSkipVerify bool) ClientOption {
```

Comment:
Are you purposefully removing the access to this option?

Reply:
I was not, thanks for catching this. Fixing it.